pandas/tests/io/test_stata.py

```diff
@@ -361,16 +361,15 @@ def test_encoding(self, version):
 
         # GH 4626, proper encoding handling
         raw = read_stata(self.dta_encoding)
-        encoded = read_stata(self.dta_encoding, encoding="latin-1")
+        encoded = read_stata(self.dta_encoding)
         result = encoded.kreis1849[0]
 
         expected = raw.kreis1849[0]
         assert result == expected
         assert isinstance(result, compat.string_types)
 
         with tm.ensure_clean() as path:
-            encoded.to_stata(path, encoding='latin-1',
-                             write_index=False, version=version)
+            encoded.to_stata(path, write_index=False, version=version)
```

Comment:
Can we make sure that the `DeprecationWarning` is issued when we pass in the `encoding` argument?